src/get-props-from-attributes.js

```diff
@@ -30,12 +30,11 @@ function getStylePropValue (attrValue) {
   })
 
   return props.reduce((props, prop) => {
-    let [propName, propValue] = prop.split(/:(.+)?/)
+    let [propName, propValue] = prop.split(/:/)
```

Comment:
This needs to be `prop.split(/:(.+)?/)` in order to support values that contain colons, e.g. `background-image: url(http://foo.bar)`. I think it makes sense for the value to be `undefined` for both `color` and `fontSize` in the case of `style="color:;font-size"`.